### DISCUSSION TITLE
LF APAC 开源布道者分享系列之四：如何去组织活动 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#29] [Announcements]
# LF APAC 开源布道者分享系列之四：如何去组织活动 - 开源之道

https://opensourceway.community/posts/lfapac-evangelist-sharing/we-need-make-event-happening/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

参考: [珠海GDG社区管理手册 ~ GDG Livin ZhuHai Life;-)](https://blog.zhgdg.org/2013-04/zhgdg-org-manual/)

其实线下/上活动的组织本身并不难,
毕竟我们生活在一个大社会中,
从小就被参加过各种活动, 从学校春游到亲戚婚礼, 
无论哪一个, 都有事先/中/后的规划以便指派执行,
嘦愿意, 任何人都可以合理复用在开源社区活动中;

只是主要问题/压力在, 组织过程中的动力问题:

- 这是什么活动? 为什么要举办?
- 大家为什么值得来?
- 举行后, 应该发布什么内容, 才能令大家不白参加?

可以简化为:

- 我为什么想组织这次活动
- 我期待有什么内容?
- 我期待的内容从哪儿可以获得?

当然, 最关键的别给自己乱定 KPI , 比如:

- 这次现场没有1000人, 我们的活动就是失败的...

俺组织的线下技术交流活动, 最冷静的一次, 台下只有两人...
最后, 一样 happy end, 有这次打底儿,
其它活动组织都不在焦虑了...

对不对?

**Reply by lijiangsheng1** [reply]
二人成行，三人成众～
